docs/overview/about.md

```diff
@@ -0,0 +1,83 @@
+# About Vald <!-- omit in toc -->
+
```

Comment:
Could you please describe this document's abstract here ?

Reply:
How about this?
```suggestion

This document gives an overview of what is Vald and what you can do with Vald. 
```

Reply:
LGTM